fiftyone/core/odm/document.py

```
@@ -797,19 +798,19 @@ def _update(
         return ops, updated_existing
 
     def _update_last_loaded_at(self):
-        self.last_loaded_at = datetime.utcnow()
+        self.last_loaded_at = datetime.now(UTC)
         self.save(virtual=True)
 
     def _update_last_modified_at(self, last_modified_at=None):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_update_last_modified_at`

Add return type annotation: `None`

(ANN202)